### PR TITLE
Add Kraken logo to PyLadies Lunch page

### PR DESCRIPTION
**Pull request by hypha**

### CODE REVIEW

File: src/content/pages/pyladies-events.mdx [hunk 1]
```diff
@@ -20,14 +24,21 @@ WHERE?
...
 
 WHEN?
 Thursday, 11th July 2024, 12:30 to 14:00.
 
 WHERE?
 Prague Congress Centre (PCC), VIP lunch area.
 
+<a href="https://kraken.tech/" target="_blank" rel="noopener noreferrer">
+    <Image src={krakenLogo} width="400" alt="Kraken Logo" />
+    <div style={{ marginBottom: '-30px' }}> </div>
```
Comment (egeakman) [inline src/content/pages/pyladies-events.mdx:37]:
```suggestion
    <Image src={krakenLogo} width="400" alt="Kraken Logo" class="-mb-30" />
```

**Review by egeakman — COMMENTED** [review]
Thanks Raquel!

File: src/content/pages/pyladies-events.mdx (hunk 1 above)
Comment (hypha) [reply src/content/pages/pyladies-events.mdx:37]: Thank you @egeakman! 

Actually I tried `class="-mb-30"`  initially, but it rendered this, which has more spacing than I wanted: 
image
 
 <div style={{ marginBottom: '-30px' }}> </div> gives me the spacing I preferred:  
 
image

Do you know why?

Comment (egeakman) [reply src/content/pages/pyladies-events.mdx:37]: Hmm, I don't know why. Astro assets normally accept classes, maybe it doesn't work well with Tailwind?

I can't look into this further right now. I think it's perfectly OK to merge it as is if it works!

Thanks again @hypha 🤗

Comment (hypha) [reply src/content/pages/pyladies-events.mdx:37]: Yeah I also thought it is not worth taking up your time to look into this now. We can talk in Prague! I will merge with thanks, @egeakman!

**Pull request merged**